60.md

```diff
@@ -0,0 +1,57 @@
+NIP-60
+======
+
+Record Events
+-------------
+
+`draft` `optional`
+
+This NIP defines a way for users to keep editable records about other people and events.
+
+## Pubkey Record
+
+A pubkey record is a kind 30082 event where the `d` tag points to the pubkey of another user.
+
+```js
+{
+  "kind": 30382,
+  "tags": [
+    ["d", "<pubkey>"],
+    ["n", "colleague"],
+    ["summary", "This is my boss"],
+    // other tags
+  ],
+  // other fields
+}
+```
+
+Any tags can be used. Specific tag uses may be formalized in other NIPs.
+
+The `content` field may be an empty string, or a NIP-44 encrypted tag list of private tags.
+
+## Event Record
+
+An event record is a kind 30083 event where the `d` tag points to the id of another event.
+
+```js
+{
+  "kind": 30383,
```

Comment:
Can you choose another number? 30383 was already used for the private relationship between two keys https://github.com/nostr-protocol/nips/pull/761

Reply:
I will change it on

Reply:
@vitorpamplona Why are you still invested in #761 when Event Sets supersedes it?
Though I used 31382 instead of 30383, it is the same thing.

Reply:
I'm already publishing kind 30383 events in Ditto.

Reply:
> @vitorpamplona Why are you still invested in #761 when Event Sets supersedes it? Although I used 31382 instead of 30383, it is the same thing.

Because you kept changing it and I needed a stable version to ship. 

Regardless, #761 is about defining all tags that can be used to clarify the relationship between two keys. It's not about just putting users into sets.

Reply:
At #784, 30382..30389 range is used for public sets, then I added 1000 to the private versions: 31382..31389.

@vitorpamplona @alexgleason you could tell me new numbers to put there instead if you think it is better.

@vitorpamplona your #761 tags are present in the event sets example. The other things like the mapping part and the obfuscating thing with sha256+hkdf are also already there at #784 to make it more generic to be used for any set of referenced items, not just the ones referencing pubkeys.